### ISSUE TITLE
Missing user in activities when switching between tabs

### ISSUE THREAD

**Issue by kamilkisiela** [kamilkisiela/graphql-hive#191]
> TypeError: Cannot read properties of undefined (reading 'displayName')

```
# src/components/v2/activities.tsx (157:18) @ getActivity
  155 | content: (
  156 |   <>
> 157 |     {user.displayName} created {targetLink} target in {projectLink} project
      |          ^
  158 |   </>
  159 | ),
  160 | icon: <PlusIcon className="h-4 w-4" />,
```

Reproduction:
1. Create an organization or visit an existing one that uses `users.displayName` in one of its activities
2. Click the "Settings" tab
3. Click the "Overview" tab
4. Error

**Comment by kamilkisiela** [comment]
The problem is that `useQuery` returns partial result, all the activities are but they all lack user.

**Comment by kamilkisiela** [comment]
Fixed by #192
